docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -11,12 +20,12 @@ ARG REF=main
 RUN git clone https://github.com/huggingface/transformers && cd transformers && git checkout $REF
 RUN python3 -m pip install --no-cache-dir -e ./transformers[dev,onnxruntime]
 
-RUN python3 -m pip install --no-cache-dir -U torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu113
+RUN python3 -m pip install --no-cache-dir -U torch==$PYTORCH torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/$CUDA
```

Comment:
With this change, `torchvision` and  `torchaudio` might get newer version.
So far, I could not find a way to get `the latest version corresponds to a specific torch version`.

Reply:
For pip to figure it out, should it be done in two separate pip installs? My gut feeling tells me yes, but I have no idea if that's actually the case

Reply:
will check it.